plugins/fileservices/fileservices.cpp

```diff
@@ -3159,7 +3159,7 @@ FILESERVICES_API bool FILESERVICES_CALL fsGetNoCommonDefault()
         Owned<IPropertyTree> env = getEnvironmentTree(daliEnv);
 
         if (env.get())
-            noCommonDef = env->getPropBool("@noCommon", noCommonDef);
+            noCommonDef = env->getPropBool("Software/Globals@noCommon", noCommonDef);
```

Comment:
@AttilaVamos did you test this?
@jakesmith Does this work?  I thought it needed to be "Software/Globals/@noCommon"

Reply:
should be "Software/Globals/@noCommon"

@AttilaVamos - it does suggest the previous PR was not adequately tested? (i.e. need to make sure an environment setting change does actually get picked up and used)

Reply:
Also, I suspect this (and similar easy to spot issues in previous PR) would be spotted by you, if you scanned the changes after you issued the PR, i.e the PR issuer should always review their own PR 1st to spot issues.
As per the devdoc/CodeReview.md document:
```
* Check for silly mistakes\
  Scan the code after creating the PR to check there are no tracing/other debugging artifacts or typos.  It doesn't take long, and often catches trivial issues.  It can avoid the need for a cycle of code-review/fixes.
```

Reply:
Yes, I tested it manually with change the value of the  "noCommon" attribute in envirionment.xml, then restart HPCC and the changed value appeared in DFU Server log.

Reply:
ok, yes I meant to say simplify by removing unecessary Owned object, didn't mean to change path used.